### DISCUSSION TITLE
Extracting FAC files through the use of an API. SAS or Python?

### DISCUSSION THREAD

**Discussion by GuyfromED** [GSA-TTS/FAC#2253] [Q&A] [answered]
The Department of Education uses FAC data to complete required audits. I was given a couple of different SAS codes (See attached) and am asking for some assistance on whether it will work or not? I have an API key, I have removed it from the code. Just not sure if it will work or not. Or, how to determine if the code works or not. 

This is beyond my current SAS knowledge, so any help would be greatly appreciated. 

[SAS API Code.txt](https://github.com/GSA-TTS/FAC/files/12730595/SAS.API.Code.txt)

**Comment by jadudm** [comment, marked as answer]
Hi @GuyfromED ,

It is possible that I replied via another comms channel to this. Also following up here just in case.

Our team has no SAS experience. We probably can cover a wide range of languages (from Java to Python to Go to Racket to Scala and more), but unfortunately, the specifics of this language and tooling are something we can only help generally with.

I would encourage you to check

https://fac.gov/developers

for the most up-to-date documentation.
